frameworks/hdfs/tools/install-secure-hdfs.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SPARK_BUILD_DIR="${DIR}/../../"
+S3_PATH=""
+HDFS_PRIMARY="hdfs"
+HTTP_PRIMARY="HTTP"
+FRAMEWORK_NAME="hdfs"
+DOMAIN="${FRAMEWORK_NAME}.autoip.dcos.thisdcos.directory"
+REALM="LOCAL"
+LINUX_USER="core"
+KEYTAB_FILE="hdfs.keytab"
+
+function add_kdc() {
+    echo "Adding kdc marathon app..."
+    # dcos marathon app add kdc.json
+
+    echo "Waiting for app to run..."
+    while true; do
+        local TASKS_RUNNING=$(dcos marathon app list --json | jq ".[0].tasksRunning")
+        if [[ ${TASKS_RUNNING} -eq 1 ]]; then
+            break
+        fi
+    done
+
+    sleep 10
+
+    SLAVE_ID=$(dcos task --json | jq -r ".[0].slave_id")
+    MASTER_PUBLIC_IP=$(curl --header "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/metadata | jq -r ".PUBLIC_IPV4")
```

Comment:
-k or --insecure is needed in the case of a non official certification. It fails without it against a test cluster.